1.0/docs/start/psk/creating-an-element.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Creating an element
+subtitle: Polymer Starter Kit
+---
+
+## Creating an element
+
```

Comment:
At present, this section is dramatically more complicated than the other sections, and more text-heavy, but most of the content doesn't have to do with PSK. This section is really about using `iron-ajax` and the books API rather than PSK itself. I think we should pull this out for now and figure out how & where best to use it.